solutions/devicesimulation-nohub/single-vm/setup.sh

```diff
@@ -1,25 +1,12 @@
 #!/usr/bin/env bash
 # Copyright (c) Microsoft. All rights reserved.
 # Note: Windows Bash doesn't support shebang extra params
-set -ex
```

Comment:
can we leave this line?

Reply:
We are logging errors from setup internal to a file, I don't think this is required to print trace of the file